flexmeasures/api/common/utils/validators.py

```diff
@@ -804,6 +808,7 @@ def decorated_service(*args, **kwargs):
                     last_asset = generic_asset
 
             # if inferred resolution is a multiple from required_solution, we can upsample_values
+            # todo: next line fails on sensors with 0 resolution
```

Comment:
Side note: underlying this seemingly small todo is a big topic of sensor resolution vs data frequency. FM works under the assumption that the frequency of posted data is some multiple of the sensor resolution, which doesn't apply to sensor measuring instantaneous values.

Reply:
I made https://github.com/SeitaBV/flexmeasures/issues/136 for this.